corehq/warehouse/models/staging.py

```diff
@@ -45,6 +48,70 @@ def clear_records(cls):
         truncate_records_for_cls(cls, cascade=False)
 
 
+class LocationStagingTable(StagingTable, CustomSQLETLMixin):
+    '''
+    Represents the staging table to dump data before loading into the LocationDim
+
+    Grain: location_id
+    '''
+    slug = LOCATION_STAGING_SLUG
+
+    domain = models.CharField(max_length=100)
+    name = models.CharField(max_length=255)
+    site_code = models.CharField(max_length=100)
+    location_id = models.CharField(max_length=255)
+    external_id = models.CharField(max_length=255, null=True)
+    supply_point_id = models.CharField(max_length=255, null=True)
+    user_id = models.CharField(max_length=255)
+
+    sql_location_id = models.IntegerField()
+    sql_parent_location_id = models.IntegerField(null=True)
+
+    location_last_modified = models.DateTimeField(null=True)
+    location_created_on = models.DateTimeField(null=True)
+
+    is_archived = models.NullBooleanField()
+
+    latitude = models.DecimalField(max_digits=20, decimal_places=10, null=True)
+    longitude = models.DecimalField(max_digits=20, decimal_places=10, null=True)
+
+    @classmethod
+    def dependencies(cls):
+        return []
+
+    @classmethod
+    def additional_sql_context(cls):
+        return {
+            'sqllocation_table': SQLLocation._meta.db_table
+        }
+
+
+class LocationTypeStagingTable(StagingTable, CustomSQLETLMixin):
```

Comment:
I kept this table de-normalized on the LocationDim because I didn't think it was worth it to normalize. Open to push back on that though, my only real deciding factor was the fact that this table was small and it reduced some complexity